src/components/common/MaintenanceLayout.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+    <div class="maintenance-page">
+        <div class="maintenance-page_content level">
+            <img
+                class="maintenance-page_img"
+                :src="require(`../../assets/img/${imageName}`)"
+                alt="Maintenance Image"
+                title="Maintenance-Image"
+            />
+            <div class="maintenance-page_description level">
+                <h1 class="maintenance-page_title">
```

Comment:
@gonzalo-plaza You can use "maintenance-page_description-title" it is more representative

Reply:
@PVillasana Here, we said of use "maintenance-page_descripiton-title", but I have read about BEM metodology and I think that the correct would be "maintenance-page_description_title", because description would be the block and the title would be the element. Is correct?